openpype/modules/clockify/clockify_module.py

```diff
@@ -218,49 +203,55 @@ def register_timers_manager(self, timer_manager_module):
     def timer_started(self, data):
         """Tell TimersManager that timer started."""
         if self._timers_manager_module is not None:
-            self._timers_manager_module.timer_started(self._module.id, data)
+            self._timers_manager_module.timer_started(self.id, data)
 
     def timer_stopped(self):
         """Tell TimersManager that timer stopped."""
         if self._timers_manager_module is not None:
-            self._timers_manager_module.timer_stopped(self._module.id)
+            self._timers_manager_module.timer_stopped(self.id)
 
     def stop_timer(self):
         """Called from TimersManager to stop timer."""
         self.clockapi.finish_time_entry()
 
     def start_timer(self, input_data):
         """Called from TimersManager to start timer."""
-        # If not api key is not entered then skip
-        if not self.clockapi.get_api_key():
-            return
 
-        actual_timer = self.clockapi.get_in_progress()
+        # get running timer to check if we need to start it
+        # TODO: Check running is not working here
         actual_timer_hierarchy = None
         actual_project_id = None
-        if actual_timer is not None:
+        # current timer does not update here
+        actual_timer = self.currently_active_timer
```

Comment:
This is based on module running in tray, which is not right. This method should be possible to execute anywhere where there is an access to the module and should always look to clockify what is the currently running timer. Storing it to module object is useless, user still can change the timer manually.